samples/dfu/updates/prepend_target_type_uart.bash

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
Could you do this from cmake instead? We typically don't use bash scripts for actions like this, instead we use cmake for the most part and use python where cmake comes short. That way you can create a target and hook this up to the build process automatically.